### PR TITLE
redirect to application list page on deleting an application

### PR DESCRIPTION
**Pull request by karthikjeeyar**
## Fixes 

https://issues.redhat.com/browse/HAC-3166

## Description

On deleting an application, we should redirect the users to the Applications list page.

## Type of change

- [x] Bugfix

## Screen shots / Gifs for design review 

https://user-images.githubusercontent.com/9964343/218961267-4b75ca41-21eb-42cf-9f7a-5c3f397c7c04.mov

## How to test or reproduce?

Delete an application from Application details page.

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [x] Safari
- [ ] Edge

### CODE REVIEW

**Review by rohitkrai03 — APPROVED** [review]
/lgtm

File: src/components/ApplicationDetails/ApplicationDetails.tsx [hunk 1]
```diff
@@ -157,7 +157,7 @@ const ApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ applicatio
               showModal<{ submitClicked: boolean }>(
                 applicationDeleteModal(application),
               ).closed.then(({ submitClicked }) => {
-                if (submitClicked) navigate('/stonesoup');
+                if (submitClicked) navigate('/stonesoup/applications');
```
Comment (karthikjeeyar) [inline src/components/ApplicationDetails/ApplicationDetails.tsx:160]: If the Workspace switcher PR goes in first then we need to updated this to `/workspaces`

**Comment by karthikjeeyar** [comment]
closing this PR in favor of https://github.com/openshift/hac-dev/pull/406

**Pull request closed without merging**